lib/filter/buddy.js

```diff
@@ -110,7 +111,19 @@ exports.update = function(json) {
       json[buddy].status_ailments.push(ailments[status].id);
       statusAilmentDesc.push(ailments[status].description);
     };
-
+    
+    /**
+     * Hit-all
+     * Overrides the record materia with Zealot and adds the Zealot Attack property
+     * Attack CAN be modified to do several things - hit more than once, etc.
+     */
+  	if (config.get('buddy.attack.multiHit')) {
+	    var materia = recordMateria["ZEALOT"];
+	    var materiaatk = recordMateria["ZEALOTATK"];
+	    json[buddy].materias = materia;
```

Comment:
The name is materias (plural) and you are assigning only zealot, I think this could break here